Terminal.Gui/Views/TreeView.cs

```diff
@@ -0,0 +1,1836 @@
+// This code is based on http://objectlistview.sourceforge.net (GPLv3 tree/list controls by phillip.piper@gmail.com).  Phillip has explicitly granted permission for his design and code to be used in this library under the MIT license.
```

Comment:
Totally minor, but can you make this comment more readable by making it fit multiple lines (so horiz scrolling isn't needed)?